shared/templates/sysctl/bash.template

```diff
@@ -22,6 +22,16 @@ for f in /etc/sysctl.d/*.conf /run/sysctl.d/*.conf /usr/local/lib/sysctl.d/*.con
   fi
 done
 
+#
+# Set sysctl config file which to save the desired value
+#
+{{% if product in [ "sle12", "sle15"] %}}
+SYSCONFIG_VAR="{{{ SYSCTLVAR }}}"
+SYSCONFIG_FILE=/etc/sysctl.d/"${SYSCONFIG_VAR//./_}".conf
```

Comment:
I think you can use Jinja `regex_replace` at build time, instead of doing the substitution at runtime with Bash.
You can also assign the escaped sysctl to a Jinja variable and reuse it.

Reply:
Thanks for the tip 🙇 will replace that

Reply:
Used basic replace but I think target is achieved and code is cleaner :+1: